[2021-03-09 12:20:37,751] {taskinstance.py:655} INFO - Dependencies all met for <TaskInstance: my_test_dag.dummy_task 2021-03-09T15:20:00.575505+00:00 [queued]>
[2021-03-09 12:20:37,756] {taskinstance.py:655} INFO - Dependencies all met for <TaskInstance: my_test_dag.dummy_task 2021-03-09T15:20:00.575505+00:00 [queued]>
[2021-03-09 12:20:37,756] {taskinstance.py:866} INFO - 
--------------------------------------------------------------------------------
[2021-03-09 12:20:37,756] {taskinstance.py:867} INFO - Starting attempt 1 of 1
[2021-03-09 12:20:37,756] {taskinstance.py:868} INFO - 
--------------------------------------------------------------------------------
[2021-03-09 12:20:37,837] {taskinstance.py:887} INFO - Executing <Task(DummyOperator): dummy_task> on 2021-03-09T15:20:00.575505+00:00
[2021-03-09 12:20:37,839] {standard_task_runner.py:53} INFO - Started process 31011 to run task
[2021-03-09 12:20:37,942] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: my_test_dag.dummy_task 2021-03-09T15:20:00.575505+00:00 [running]> rep-pc
[2021-03-09 12:20:37,951] {taskinstance.py:1048} INFO - Marking task as SUCCESS.dag_id=my_test_dag, task_id=dummy_task, execution_date=20210309T152000, start_date=20210309T152037, end_date=20210309T152037
[2021-03-09 12:20:47,665] {logging_mixin.py:112} INFO - [2021-03-09 12:20:47,665] {local_task_job.py:103} INFO - Task exited with return code 0
